### ISSUE TITLE
Add Triangle and Tetrahedron element (cell) classes

### ISSUE THREAD

**Issue by adtzlr** [adtzlr/felupe#4]
As discussed in [here](#3),

- [x] add support for linear & quadratic Triangle (Tri1, Tri2) and Tetrahedron (Tet1, Tet2) elements

progress:
- [x] Tri1
- [x] Tri2
- [x] Tet1
- [x] Tet2

**Status**: experimental

**Comment by adtzlr** [comment]
For linear tets (`Tet1`), an example is given here:

```python
import numpy as np
import felupe as fe

import meshzoo
nodes, connectivity = meshzoo.cube_tetra((0,0,0), (1,1,1), n=10)

connectivity_fixed = connectivity.copy()
xx = nodes[connectivity][:,1] - nodes[connectivity][:,0]
yy = nodes[connectivity][:,2] - nodes[connectivity][:,0]
zz = nodes[connectivity][:,3] - nodes[connectivity][:,0]
det = np.diag(np.dot(zz, np.cross(xx, yy).T))
connectivity_fixed[np.where(det < 0)] = connectivity_fixed[np.where(det < 0)][:,[0,2,1,3]]

mesh = fe.mesh.Mesh(nodes, connectivity_fixed, etype="tetra")
element = fe.element.Tet1()

import quadpy
scheme = quadpy.t3.schemes["hammer_marlowe_stroud_1"]()
tetra = np.array([[0,0,0],
                  [1,0,0],
                  [0,1,0],
                  [0,0,1]])
points = quadpy.tn.transform(scheme.points, tetra.T).T
weights = scheme.weights

quadrature = fe.quadrature.Scheme(points, weights)
```

**Comment by adtzlr** [comment]
closed; see #9 and pull request a2d588538bfdc76c5a333de694da2f4e706dff39
